pkg/cmd/server/start/start_allinone.go

```diff
@@ -91,11 +94,12 @@ func NewCommandStartAllInOne() (*cobra.Command, *AllInOneOptions) {
 
 	flags := cmd.Flags()
 
-	flags.BoolVar(&options.WriteConfigOnly, "write-config", false, "Indicates that the command should build the configuration from command-line arguments, write it to the locations specified by --master-config and --node-config, and exit.")
```

Comment:
Can we prevent `openshift.local.config` from displaying as a default in the help?